ToolBox/FxcmDownloader/FxcmDataDownloader.cs

```diff
@@ -163,35 +161,39 @@ public IEnumerable<BaseData> Get(Symbol symbol, Resolution resolution, DateTime
             // initialize session
             RequestTradingSessionStatus();
 
-            Console.WriteLine("Downloading data from {0} to {1}...", startUtc.ToShortDateString(), endUtc.ToShortDateString());
+            Console.WriteLine("Downloading {0} data from {1} to {2}...", Enum.GetName(typeof(Resolution), resolution), startUtc.ToShortDateString(), endUtc.ToShortDateString());
```

Comment:
The expression `Enum.GetName(typeof(Resolution), resolution)` could be simplified as just `resolution`.